Sources/POSIX/stat.swift

```diff
@@ -10,14 +10,34 @@
 
 import libc
 
+/// A structure containing information about a file.
+public struct FileStatus {
```

Comment:
I agree but I have mixed feelings about this one. POSIX is to make things "swifty" and platform independent which kinda fits here. But ok.